### PR TITLE
add simple cwl for bidi and xepersian packages

### PR DESCRIPTION
**Pull request by hadisfr**
bidi 33.0 – Bidirectional typesetting in plain TEX and LATEX, using XETEX: https://ctan.org/pkg/bidi
xepersian 21.2 – Persian for LATEX, using XETEX: https://ctan.org/pkg/xepersian

### CODE REVIEW

File: completion/xepersian.cwl [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+\today
+\latintoday
+\twocolumnstableofcontents
+\XePersian
+\plq
+\pr
+\Latincite
+\harfinumeral{integer}
+\adadinumeral{integer}
+\tartibinumeral{integer
```
Comment (timhoffm) [inline completion/xepersian.cwl]: Closing brace missing

Comment (hadisfr) [reply completion/xepersian.cwl]: I added some `*`s. 😀

**Comment by hadisfr** [comment]
I added some `*`s. 😀

**Comment by sunderme** [comment]
still some closing braces are missing ...

File: completion/xepersian.cwl (hunk 1 above)
Comment (hadisfr) [reply completion/xepersian.cwl]: done!

**Comment by sunderme** [comment]
\tartibinumeral{integer  ?

**Comment by hadisfr** [comment]
didn't sync my local files with repo. :| Sorry.

**Comment by thatlittleboy** [comment]
I think there are still some typos in bidi.cwl?

`\LTRfootnotetex[num]{text}`

`\RTLdblco` 

`\debugfootnotedirectio`

**Comment by hadisfr** [comment]
double checked with packages' doc.
+ failure of travis-ci... not my fault. :-"

**Comment by sunderme** [comment]
thanks

**Comment by hadisfr** [comment]
You're welcome! :D

**Pull request merged**